docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,92 @@
 
 ## Using Tinylicious locally
 
+You can run tinylicious locally by executing the following command,
+
+```sh
+npx tinylicious
+```
+
+By default, Tinylicious runs on port `7070`, however you can change port by navigating to `server/tinylicious` and running the below command on `Windows Powershell` by specifying the port number,
+
+```sh
+$env:PORT=6502
+npm run start
+```
+
+Now, you can navigate to `http://localhost:6502` on your browser to see Tinylicious service up and running.
+
 ## How to deploy using Tinylicious
 
+The `AzureClient` supports both instantiating against a deployed Azure Fluid Relay service instance for production scenarios, as well as against a local, in-memory service instance, known as Tinylicious, for development purposes.
+
+You can connect to a live Azure Fluid Relay instance by passing in the tenant ID as `local`, the orderer and storage URLs  pointing to the Tinylicious instance on the default values of `http://localhost:7070` to connect to a local Tinylicious server for development purposes. You can use `InsecureTokenProvider` for token resolution while running the service locally for development purposes.
+
+In the below code snippet, `AzureClient` is pointing to Tinylicious service. This is providing you with an added feature to work with both the local Tinylicious service and the deployed Azure Fluid Relay service. `TinyliciousClient` only works with local Tinylicous service.
+
+```typescript
+import { AzureFunctionTokenProvider, AzureClient, AzureConnectionConfig } from "@fluidframework/azure-client";
+import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+
+const user = {
+    id: "UserId",
+    name: "Test User",
+};
+
+const connectionConfig: AzureConnectionConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("fooBar", user),
+    // if you're running Tinylicious on a non-default port, you'll need change these URLs
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
+
+const client = new AzureClient(connectionConfig);
+```
+
+To launch the local Tinylicious service instance, run `npx tinylicious` from your terminal window.
+
 ## Using custom domains with Tinylicious
+
+You can use `ngrok` which lets you expose port on your local machine to the internet. Ngrok gives you a random hostname for each tunnel. This tool enables you to attach domains/subdomains against those tunnels.
+
+1. Navigate to [ngrok](https://ngrok.com/) and Sign-up. There would be an `AUTH_TOKEN` generated on signing up.
+
+2. Download [ngrok](https://ngrok.com/download) and unzip the folder
+
+3. Connect to your account by running the following command,
+
+```sh
+ngrok authtoken AUTH_TOKEN
+```
+
+Running this command will add your authtoken to the default ngrok.yml configuration file.
+
+4. Run Tinylicious service locally
+
+```sh
+npx tinylicious
+```
+
+5. Fire ngrok
```

Comment:
Let's move the instructions about what `PORT_NUMBER` would be up to here so that readers know what to expect to fill out for the command before they read the command